core/kazoo_auth/priv/couchdb/views/providers.json

```diff
@@ -0,0 +1,13 @@
+{
+    "_id": "_design/providers",
```

Comment:
Could you move `enabled_providers_by_type` and `providers_by_type` from `auth.json` here and update `cb_multi_factor`?